database/models/sensors.js

```diff
@@ -101,18 +109,24 @@ module.exports = {
     },
 
     delete: function(id) {
-        return databaseP.then(function (db) {
+        return databaseP
+        .then(function (db) {
             
-            var query = sensors
-                .delete()
-                .where(sensors.sim.equals(id))
-                .returning('*')
-                .toQuery();
-
-            return new Promise(function (resolve, reject) {
-                db.query(query, function (err, result) {
-                    if (err) reject(err);
-                    else resolve(result.rows[0]);
+            // Delete related measurements
+            return databaseMeasurements.deleteById(id)
```

Comment:
Careful, this file is for pure `sensor` functions.
MultiTable functions should be in the `index.js` file.
But anyways, i guess we shouldn't be able to delete all measurements that easily. We should prompt a message that warns the user that measurements are about to be deleted also.